src/deque.py

```diff
@@ -0,0 +1,45 @@
+# _*_ encoding: utf-8 _*_
+from queue import Queue
+
+
+class Deque(object):
+    """Make deque object."""
+    def __init__(self, val=None):
+        """Make new deque."""
+        self._container_q = Queue(val)
+
+    def pop(self):
+        """Pop item from end of deque."""
+        return self._container_q._container.shift()
```

Comment:
What happens when you try this pop method on a queue with one item in it? Write tests for edge cases. I'm getting an error on a 1 item list, but the `pop_left()` works.